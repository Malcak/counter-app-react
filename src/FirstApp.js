import React from 'react';

const FirstApp = () => {
  return (
    <>
      <h1>Hola mundo</h1>
      <p>Mi primera aplicaci√≥n</p>
    </>
  );
};

export default FirstApp;
