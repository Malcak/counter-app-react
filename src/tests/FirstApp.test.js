import React from 'react';
import { shallow } from 'enzyme';
// import { render } from '@testing-library/react';

import FirstApp from '../FirstApp';

describe('Test in <FirstApp /> component', () => {
  // test with jest dom extend expect
  // test('should show the message "Hola, soy Goku"', () => {
  //   const saludo = 'Hola, soy Goku';
  //   const { getByText } = render(<FirstApp saludo={saludo} />);
  //   expect(getByText(saludo)).toBeInTheDocument();
  // });

  // test with enzyme
  test('should display the <FirstApp /> component correctly', () => {
    const saludo = 'Hola, soy Goku';
    const wrapper = shallow(<FirstApp saludo={saludo} />);
    expect(wrapper).toMatchSnapshot();
  });
});
