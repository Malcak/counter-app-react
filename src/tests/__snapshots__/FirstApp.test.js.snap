// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test in <FirstApp /> component should display the <FirstApp /> component correctly 1`] = `
<Fragment>
  <h1>
    Hola, soy Goku
  </h1>
  <p>
    Mi primera aplicaci√≥n
  </p>
</Fragment>
`;
